src/compat/db_query.c

```diff
@@ -2455,8 +2455,8 @@ int
 db_query_seek_tuple (DB_QUERY_RESULT * result, int offset, int seek_mode)
 {
   int scan;
-  int rel1, rel2, rel3, rel_n;
-  int curr_tplno, tpl_cnt;
+  INT64 rel1, rel2, rel3, rel_n;
```

Comment:
rel1 is assigned from offset, but the offset is not a INT64 type.
and the offset is passed from fn_execute_internal's max_row which's type is int not INT64.
max_row type is from client and the type should be change to INT64 first.